gains: [
  "optic_flow",
  "gps",
]

optic_flow:

  horizontal:
    kp: 0.15
    kv: 0.15
    ka: 1.0

    kib: 0.01
    kib_lim: 0.15

    kiw: 0.01
    kiw_lim: 0.3

  vertical:
    kp: 0.6
    kv: 0.2
    ka: 1.0

  weight_estimator:
    km: 1.0
    km_lim: 1.0

gps:

  horizontal:
    kp: 0.60
    kv: 0.30
    ka: 1.0

    kib: 0.01
    kib_lim: 0.15

    kiw: 0.01
    kiw_lim: 0.3

  vertical:
    kp: 0.6
    kv: 0.2
    ka: 1.0

  weight_estimator:
    km: 1.0
    km_lim: 1.0 # [kg]
